src/countries/id.js

```diff
@@ -10,21 +10,43 @@ var config = {
   },
 }
 
-var fixPhoneShort = tools.fixPhoneBuilder(10, 13, config);
-var fixPhoneLong = tools.fixPhoneBuilder(10, 14, config);
+var longPhoneConfig = {
+  countryCode: '+62',
+  countryLocalPrefix: '',
+  localCodeLength: 3,
+  phoneLength: 8,
+  hasLocalPrefix: function (phone) {
+    return false
+  },
+}
+
+var fixPhoneShort = tools.fixPhoneBuilder(10, 13, shortPhoneConfig);
+var fixPhoneLong = tools.fixPhoneBuilder(10, 14, longPhoneConfig);
 
 var fixIdPhone = function (phone) {
   phone = phone.replace(/^(0{1,2})/, '')
   phone = phone.replace(/^\+/, '')
-  var localCodeLength = tools.getLocalCode(config, phone).length
+  var localCodeLength = tools.getLocalCode(shortPhoneConfig, phone).length
   if (localCodeLength === 4) {
     return fixPhoneLong(phone)
   } else {
     return fixPhoneShort(phone)
   }
 }
 
+var decomposeLongPhone = tools.decomposeBuilder(fixIdPhone, longPhoneConfig)
+var decomposeShortPhone = tools.decomposeBuilder(fixIdPhone, shortPhoneConfig)
+
+function decomposeBuilder (phone) {
```

Comment:
builder is a function which returns function
is this function returning a function?
rename it